package.json

```diff
@@ -9,6 +9,8 @@
   "dependencies": {
     "dexie": "^1.2.0",
     "dragula": "^3.6.2",
+    "electron-installer-debian": "^3.1.0",
+    "electron-installer-redhat": "^3.2.0",
```

Comment:
These are already in optionalDependencies, and they need to stay there, because they aren't (weren't?) installable on some OS's.

Reply:
I don't remember changing that but 1 second

Reply:
Fixed.